packages/core/android/src/main/kotlin/com/datadog/reactnative/DatadogSDKWrapper.kt

```diff
@@ -21,6 +26,45 @@ import com.datadog.android.trace.Trace
 import com.datadog.android.trace.TraceConfiguration
 import com.datadog.android.webview.WebViewTracking
 
+/**
+ * Internal object used to add internal testing.
+ */
+object DatadogSDKWrapperStorage {
+    internal val onInitializedListeners: MutableList<(InternalSdkCore?) -> Unit> = mutableListOf()
+
+    /**
+     * Adds a Listener called when the core is initialized.
+     */
+    fun addOnInitializedListener(listener: (InternalSdkCore?) -> Unit) {
```

Comment:
Does it make sense to call listener with `null` as an argument? Maybe the type should be changed to `(InternalSdkCore) -> Unit` instead?